fishcore/ClientApp/src/components/DevelopmentWindow.js

```diff
@@ -0,0 +1,69 @@
+﻿import React, { useState, useRef } from 'react';
```

Comment:
The DevelopmentWindow currently uses the example code from [React Example](https://reactflow.dev/examples/drag-and-drop/)